cirq/contrib/svg/svg.py

```diff
@@ -60,45 +62,104 @@ def _fit_horizontal(tdd: 'cirq.TextDiagramDrawer',
     return col_starts, col_widths
 
 
+def _fit_vertical(tdd: 'cirq.TextDiagramDrawer',
+                  ref_boxheight: float, row_padding: float) \
+        -> Tuple[List[float], List[float], Dict[float, int]]:
+    # Note: y values come as half integers. Map to integers
+    all_yis = sorted({yi for _, yi in tdd.entries.keys()}
+                     | {yi1 for _, yi1, _, _ in tdd.vertical_lines}
+                     | {yi2 for _, _, yi2, _ in tdd.vertical_lines}
+                     | {yi for yi, _, _, _ in tdd.horizontal_lines})
+    yi_map = {yi: i for i, yi in enumerate(all_yis)}
+
+    max_yi = max(yi_map[yi] for yi in all_yis)
```

Comment:
Does this blow up when given the empty circuit?

Reply:
yes

Reply:
added a better error message for this and the other failure case (empty moments). I don't want to figure out how to fix it (particularly the latter)